### ISSUE TITLE
Tensorflow 3D demos?

### ISSUE THREAD

**Issue by patrickoshaughnessey** [google-research/google-research#637]
I am unable to find a link to the demos mentioned int he setup doc:

https://github.com/google-research/google-research/blob/master/tf3d/doc/setup.md

"Now you are ready to start training or evaluation or try one of our **demos**"

Can you add a link, please?

**Comment by FrancescoMandru** [comment]
Yes I would like to better understand the usage of this repository. For example, suppose that I have a dataset of point clouds with the related labels. (Eg Semantic Kitti dataset) and I just want to exploit the modules for 3D sparse convolutions, or the U-Net semantic segmentation model which are the core functions that are useful to try. How should I prepare my dataset? How can I build a simple model to train and test the semantic segmentation results? An `helloworld` example would be fantastic!
